Makefile

```diff
@@ -384,43 +384,47 @@ telemetrygenlite:
 # helper function to update the core packages in builder-config.yaml
 # input parameters are
 # $(1) = path/to/versions.yaml (where it greps the relevant packages)
-# $(2) = path/to/go.mod (where it greps the package-versions)
-# $(3) = path/to/builder-config.yaml (where we want to update the versions)
+# $(2) = path/to/builder-config.yaml (where we want to update the versions)
 define updatehelper
-	if [ ! -f $(1) ] || [ ! -f $(2) ] || [ ! -f $(3) ]; then \
-			echo "Usage: updatehelper <versions.yaml> <go.mod> <builder-config.yaml>"; \
-			exit 1; \
+	if [ ! -f $(1) ] || [ ! -f $(2) ]; then \
+		echo "Usage: updatehelper <versions.yaml> <builder-config.yaml>"; \
+		exit 1; \
 	fi
+	# For each line in versions.yaml referencing go.opentelemetry.io,
+	# try to find it in any go.mod under the current directory.
 	grep "go\.opentelemetry\.io" $(1) | sed 's/^[[:space:]]*-[[:space:]]*//' | while IFS= read -r line; do \
-			if grep -qF "$$line" $(2); then \
-					package=$$(grep -F "$$line" $(2) | head -n 1 | awk '{print $$1}'); \
-					version=$$(grep -F "$$line" $(2) | head -n 1 | awk '{print $$2}'); \
-					builder_package=$$(grep -F "$$package" $(3) | awk '{print $$3}'); \
-					builder_version=$$(grep -F "$$package" $(3) | awk '{print $$4}'); \
-					if [ "$$builder_package" == "$$package" ]; then \
-						sed -i.bak -e "s|$$builder_package.*$$builder_version|$$builder_package $$version|" $(3); \
-						rm $(3).bak; \
-						echo "[$(3)]: $$package updated from $$builder_version to $$version"; \
-					fi; \
+		found="false"; \
+		for gomod in $$(find . -name 'go.mod'); do \
+			if grep -qF "$$line" "$$gomod"; then \
+				found="true"; \
+				package=$$(grep -F "$$line" "$$gomod" | head -n 1 | awk '{print $$1}'); \
```

Comment:
Not sure if this work, because for example no other module depends on zpages.

Reply:
Yeah, I figured that as well after trying this out